manifests/n/NCHSoftware/Crescendo/8.28+/NCHSoftware.Crescendo.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: NCHSoftware.Crescendo
+PackageVersion: 8.28+
+PackageLocale: en-US
+Publisher: NCH Software
+PackageName: Crescendo

```

```suggestion
PackageName: Crescendo Music Notation Editor
```
This must match Control Panel.